apps/verticalhome/build/default-homescreens.json

```diff
@@ -24,6 +24,6 @@
     ]
   ],
   "preferences": {
-    "grid.cols": 3
+    "grid.cols": 2
```

Comment:
Given that the number of columns will not be configurable, I think that it doesn't need to be in this JSON file anymore.  Maybe find the code that reads this file and just define a const for the number of columns.